Sample Applications/DataBindingDemo/Styles.xaml

```diff
@@ -8,36 +8,79 @@
         <Setter Property="FontSize" Value="18" />
     </Style>
 
+    <Style x:Key="GridStyle" TargetType="Grid" >
+        <Setter Property="Background" Value="Transparent" />
+        <Setter Property="Label.FontWeight" Value="Bold" />
+    </Style>
+
+    <Style x:Key="SmallTitleAlignmentStyle" TargetType="TextBlock">
+        <Setter Property="VerticalAlignment" Value="Center" />
+        <Setter Property="HorizontalAlignment" Value="Right" />
+    </Style>
+
     <Style x:Key="SmallTitleStyle" TargetType="TextBlock">
-        <Setter Property="FontWeight" Value="Bold" />
-        <Setter Property="Foreground" Value="DimGray" />
+        <Setter Property="Label.FontWeight" Value="Bold" />
```

Comment:
This is targeting TextBlock, but you're setting Label.FontWeight.  Is this a copy/paste issue?  You can just say FontWeight and it should be fine.

Reply:
Corrected the property to just fontweight